src/test/java/com/stripe/model/EventTest.java

```diff
@@ -31,11 +31,11 @@ public void serializesToJson() throws IOException {
 
 		assertEquals(reserializedEvent.getId(), event.getId());
 		assertEquals(reserializedEvent.getObject(), event.getObject());
+		assertEquals(reserializedEvent.getAccount(), event.getAccount());
```

Comment:
Update: that test doesn't even support the old `user_id` key so I guess we're good though testing it explicitly might be worth it.